docs/release-notes/template.rst

```diff
@@ -0,0 +1,4 @@
+:Type: Kickstart / TUI / GUI / Modularization / ...
+:Summary: Summarize the change (#bug number)
+:Description: Describe the change in detail.
+:Link: Provide links to pull requests, bugs or issues.
```

Comment:
I'm not sure if more links need more `:Link:` lines, but I imagine that's irrelevant because the result will be processed by human anyway.

Reply:
So maybe change this to `:Links:` and add info that there could be multiple links each on one line?

Reply:
Fixed.